requirements.txt

```diff
@@ -1,4 +1,5 @@
-M2Crypto < 1.0.0
+cryptography < 2.0.0
 nose < 2.0.0
 pep8 < 2.0.0
 pyflakes < 1.0.0
+six < 2.0.0
```

Comment:
Six is a dependency of the cryptography library, so we're not saving anything here, but I went ahead and removed it anyway in off chance we ever change crypto libs again.